tests/x11/nis_client.pm

```diff
@@ -17,31 +17,37 @@ use warnings;
 use testapi;
 use lockapi 'mutex_lock';
 use utils;
-use version_utils 'is_sle';
+use version_utils qw(is_opensuse is_sle);
 use mm_network 'setup_static_mm_network';
 use y2_module_guitest '%setup_nis_nfs_x11';
 use x11utils 'turn_off_gnome_screensaver';
 use y2_module_consoletest;
 
 sub setup_nis_client {
-    if (is_sle('>=15')) {
-        assert_screen 'nis-client-install-missing-package';
-        send_key 'alt-i';    # install missing ypbind rpm
-    }
+    assert_screen 'nis-client-install-missing-package';
+    send_key 'alt-i';    # install missing ypbind rpm
     assert_screen 'nis-client-configuration', 120;
-    send_key 'alt-u';        # use NIS radio button
+    send_key 'alt-u';    # use NIS radio button
     wait_still_screen 4;
-    send_key 'alt-l';        # open firewall port
+    send_key 'alt-l';    # open firewall port
     assert_screen 'nis-client-fw-opened';
-    send_key 'alt-i';        # NIS domain
+    send_key 'alt-i';    # NIS domain
     type_string $setup_nis_nfs_x11{nis_domain};
-    send_key 'alt-m';        # start automounter
+    send_key 'alt-m';    # start automounter
     wait_still_screen 4;
-    send_key 'alt-d';        # find NIS server
-    assert_screen 'nis-client-server-in-domain', 120;
-    send_key 'spc';          # select found NIS server
-    assert_screen 'nis-client-server-in-domain-selected';
-    send_key 'alt-o';        # OK
+    send_key 'alt-d';    # find NIS server
+    assert_screen [qw(nis-client-server-in-domain nis_servers_in_domain_empty)], 120;
+    if (match_has_tag 'nis_servers_in_domain_empty') {
+        send_key 'alt-o';
+        send_key 'alt-a';
+        type_string '10.0.2.1';
```

Comment:
I did not use $setup_nis_nfs_x11{server_address} here, as it looked like 10.0.2.1 so I would have needed to remove that /18. I did not see the point as that value is hard-coded anyway in y2_module_guitest.

Reply:
but it is better to adjust it the y2_module_guitest to sustain consistency

Reply:
if something changes, it would require changes in more than one place, just using `my ($ip_no_mask, $mask) = split('/', $ip);` in the same way than in the internal method: `configure_static_ip` should be better.

Reply:
I found that ther was already this (my $server_ip = $setup_nis_nfs_x11{server_address}) =~ m/\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}/; but you solution looks better